QUICKSTART.md

```diff
@@ -43,15 +68,15 @@ monitored. More information is available in the specification chapter [Obtaining
 
 ```java
 Tracer tracer =
-    OpenTelemetry.getTracer("instrumentation-library-name","semver:1.0.0");
+    openTelemetry.getTracer("instrumentation-library-name","semver:1.0.0");
 ```
 
 ### Create basic Span
 To create a basic span, you only need to specify the name of the span.
 The start and end time of the span is automatically set by the OpenTelemetry SDK.
 ```java
 Span span = tracer.spanBuilder("my span").startSpan();
-try (Scope scope = TracingContextUtils.currentContextWith(span)) {
+try (Scope scope = tracer.withSpan(span)) {
```

Comment:
`tracer.withSpan` is gone. This should be `Context.with(span).makeCurrent()`, right?

Reply:
And something else soon - I'm taking this change out of this PR anyways since we'd want to do it with the next release.